metamist/parser/sample_file_map_parser.py

```diff
@@ -77,16 +96,21 @@
 class SampleFileMapParser(GenericMetadataParser):
     """Parser for SampleFileMap"""
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
```

Comment:
Disabling 'too-many-arguments', looks like we might need to do a bit of refactoring. 
Just an idea, feel free to ignore :-)
What if all the Sequencing is wrapped in its own class, so it contains type, technology, platform, facililty, library and all the defaults.

Reply:
@milo-hyben thanks for the suggestion, sorry to take a few weeks to come back to this. 

I added a class in `generic_parser.py` called `DefaultSequencing`, which holds `seq_type`, `technology`, `platform`, `facility`, and `library`, and I updated the parsing modules and tests to use this class when entering these sequencing fields.